lib/taurus/qt/qtgui/graphic/taurusgraphic.py

```diff
@@ -39,6 +39,8 @@
 
 import Queue
 
+from PyQt4 import Qt
```

Comment:
Me too, in fact I've it removed in my local file ( from taurus.external.qt import Qt is already in the file ) ... I don't know how it ended up here ...

Thanks!